.vib/jenkins/cypress/cypress/integration/jenkins_spec.js

```diff
@@ -0,0 +1,94 @@
+/// <reference types="cypress" />
+import { random, checkErrors } from './utils';
+
+it('allows user to log in and log out', () => {
+  cy.login();
+  cy.contains('.alert-danger').should('not.exist');
+  cy.get('div#tasks');
+
+  cy.get('[href$="/logout"]').click();
+  cy.get('form[name*="login"]');
+});
+
+it('should be possible to create a new Git Project', () => {
+  cy.login();
+  cy.visit('/view/All/newJob');
+
+  cy.fixture('items').then((item) => {
+    cy.get('#name').type(`${item.freestyleProject.name}-${random}`);
+    cy.contains(item.freestyleProject.type).click();
+    cy.contains('button', 'OK').should('be.enabled').click();
+    cy.contains('Error').should('not.be.visible');
+
+    cy.get('.radio-block-start').contains('Git').click();
+    cy.get("input[name*='url']").type(item.freestyleProject.repositoryURL);
+    cy.contains('button', 'Save').click();
+    cy.contains('h1', item.freestyleProject.name);
+  });
+
+  // As a new Git project is created in every execution, the next build
+  // will always be the first one.
+  const nextBuildNumber = 1;
+  cy.get("a[title='Build Now']").click();
+  cy.contains(`#${nextBuildNumber}`, { timeout: 30000 });
```

Comment:
Instead of having custom command timeouts, I'd recommend increasing the default timeout to the maximum value you expect.
Cypress will wait until then. 
`"defaultCommandTimeout":120000,`

Do not worry about slowing down the test. That is the maximum time to wait, not the default one.

Reply:
Okay, no strong thoughts about this so I will apply your suggestion 😁